packages/credential-ld/src/suites/Ed25519Signature2020.ts

```diff
@@ -38,8 +39,9 @@ export class VeramoEd25519Signature2020 extends VeramoLdSignature {
           keyRef: key.kid,
           data: messageString,
           encoding: 'base64',
+          algorithm: 'EdDSA',
         })
-        return u8a.fromString(signature)
+        return base64ToBytes(signature)
```

Comment:
This kind of bug should not make it into our codebase..
We'll be much stricter about tests for new code going forward